bin/watch.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
```

Comment:
I'm assuming you intended to rename https://github.com/replicatedhq/troubleshoot/blob/main/bin/watchrsync.js?

Reply:
forgot to remove it from the PR